CDCgov/prime-reportstream
# 13136: metrics part 2

This PR adds a new event `ReportAcceptedEvent` which is emitted when a report is ready to be broken down by receiver.

Test Steps:
1. Run the application locally with the application insights jar and properly set up environment variable
2. Run the smoke tests
3. See events in the Application Insights dashboard

## Changes
- New event
- Added parent report ID to other event to facilitate better querying

## Checklist

### Testing

## Linked Issues
- Fixes #13136
